cnf-certification-test/identifiers/identifiers.go

```diff
@@ -1465,7 +1465,7 @@ tag. (2) It doesn't have any of the following prefixes: default, openshift-, ist
 			NonTelco: Optional,
 			Extended: Mandatory,
 		},
-		TagCommon, TagTelco)
+		TagCommon)
```

Comment:
Should this be `TagExtended` not `TagCommon`?  Since everything is optional except for `Extended` test suite?

Reply:
fixed